avocado/plugins/jobs.py

```diff
@@ -0,0 +1,86 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2020
+# Authors: Beraldo Leal <bleal@redhat.com>
+
+"""
+Jobs subcommand
+"""
+import json
+import os
+
+from datetime import datetime
+from glob import glob
+
+from avocado.core import exit_codes
+from avocado.core.data_dir import get_logs_dir
+from avocado.core.plugin_interfaces import CLICmd
+
+
+class Jobs(CLICmd):
+    """
+    Implements the avocado 'jobs' subcommand
+    """
+    name = 'jobs'
+    description = 'Manage Avocado jobs'
+
+    def configure(self, parser):
+        """
+        Add the subparser for the assets action.
+
+        :param parser: The Avocado command line application parser
+        :type parser: :class:`avocado.core.parser.ArgumentParser`
+        """
+        parser = super(Jobs, self).configure(parser)
+
+        subcommands = parser.add_subparsers(dest='jobs_subcommand')
```

Comment:
Suggestion: `metavar=command`, or `metavar="jobs subcommand"`. It shows a better help message and error message.

Reply:
@clebergnu yes sure. My only suggestion is to use `metavar='sub-command'`, since that the command is already `jobs`.